GwApiService/src/main/java/us/gameandwatching/gwapi/service/calc/EloCalculator.java

```diff
@@ -0,0 +1,81 @@
+package us.gameandwatching.gwapi.service.calc;
+
+import com.google.inject.Inject;
+import com.google.inject.Singleton;
+
+import us.gameandwatching.gwapi.core.EloPair;
+import us.gameandwatching.gwapi.core.MatchResult;
+
+@Singleton
+public class EloCalculator {
+
+  @Inject
+  public EloCalculator() {
```

Comment:
If it's static then testing classes that utilize it becomes much more difficult.  I left the constructor there to be explicit, but I can remove it until we need it